server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetMailboxesResponse.java

```diff
@@ -31,10 +33,12 @@ public static Builder builder() {
 
     public static class Builder {
 
-        private ImmutableMap.Builder<MailboxCreationId, Mailbox> created;
+        private final ImmutableMap.Builder<MailboxCreationId, Mailbox> created;
+        private final ImmutableMap.Builder<MailboxCreationId, SetError> notCreated;
 
         private Builder() {
             created = ImmutableMap.builder();
+            notCreated = ImmutableMap.builder();
         }
 
         public Builder creation(MailboxCreationId creationId, Mailbox mailbox) {
```

Comment:
This is fixed in this commit:
https://github.com/linagora/james-project/pull/202/commits/e1cae25134d1bbb897e6c2d7186ddca392e01796